csharp/ql/src/semmle/code/csharp/controlflow/internal/Splitting.qll

```diff
@@ -14,7 +14,7 @@ private import ControlFlow
 private import SuccessorTypes
 
 /** The maximum number of splits allowed for a given node. */
-private int maxSplits() { result = 7 }
+private int maxSplits() { result = 5 }
```

Comment:
How was this number derived?

Reply:
Changed one random number to another ;-) At a customer, 7 proved to be too high, lowering it to 5 worked.